.github/workflows/dependencies/clang15.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 The WarpX Community
```

Comment:
Should this be BLAST instead of WarpX from now on?

Reply:
You are right